scripts/generate-framework-agreement-signature-pages.py

```diff
@@ -61,19 +63,17 @@
     args = docopt(__doc__)
 
     framework_slug = args['<framework_slug>']
-    client = DataAPIClient(get_api_endpoint_from_stage(args['<stage>']), args['<api_token>'])
+    client = DataAPIClient(get_api_endpoint_from_stage(args['<stage>']), get_auth_token('api', args['<stage>']))
     framework = client.get_framework(framework_slug)['frameworks']
     framework_lot_slugs = tuple([lot['slug'] for lot in client.get_framework(framework_slug)['frameworks']['lots']])
-    framework_kwargs = framework['frameworkAgreementDetails']
-    framework_kwargs['frameworkName'] = framework['name']
 
     supplier_id_file = args['<supplier_id_file>']
     supplier_ids = get_supplier_ids_from_file(supplier_id_file)
     html_dir = tempfile.mkdtemp()
 
     records = find_suppliers_with_details_and_draft_service_counts(client, framework_slug, supplier_ids)
     headers, rows = get_csv_rows(records, framework_slug, framework_lot_slugs, count_statuses=("submitted",))
-    render_html_for_successful_suppliers(rows, framework_kwargs, args['<template_folder>'], html_dir)
+    render_html_for_successful_suppliers(rows, framework, 'templates/framework_signature_page', html_dir, client)
```

Comment:
Feels a bit weird having the template dir hardcoded here. Had a _quick_ look and couldn't see this method being used anywhere other than `dmscripts/generate_framework_agreement_signature_pages.py`. Could just hardcode it there?

Reply:
Also, the client being passed in here is never used.

Reply:
This script is the entrypoint for the real python functions that do the work - so it's basically configuration and setup. This feels like the higher level place to hardcode a directory path, if anywhere, rather than deep inside the script.

Yeah the client is a good spot. Was originally going to add an extra call to get the supplier_framework inside the script but it wasn't needed. Have removed.